locale/en/manageiq.po

```diff
@@ -39401,11 +39402,8 @@ msgid "Invalid authentication credentials"
 msgstr ""
 
 #: ../app/controllers/mixins/generic_button_mixin.rb
-msgid "Invalid button action"
-msgstr ""
-
 #: ../app/controllers/service_controller.rb
-msgid "Invalid button action!"
+msgid "Invalid button action"
```

Comment:
In addition to this fix, I opened https://github.com/ManageIQ/manageiq-ui-classic/pull/9044 to normalize "invalid button" action to also use this msgid